requirements.txt

```
@@ -1,6 +1,6 @@
 # All things django and asyncio are deliberately left to pulpcore
 # Example transitive requirements: asgiref, asyncio, aiohttp
-pulpcore>=3.49.0,<3.55
+pulpcore>=3.55.0,<3.70
```

I don't think we need to raise the floor, just the ceiling?
```suggestion
pulpcore>=3.49.0,<3.70
```

---

pulp_rpm just raised the upper bound as well.